README.md

```diff
@@ -881,6 +881,29 @@ interface FCPAttribution {
 #### `INPAttribution`
 
 ```ts
+/**
+ * Summary information about the slowest script intersecting the INP duration
```

Comment:
```suggestion
 * Summary information about the longest script intersecting the INP duration
```

If we're going to call it "longest" script in the API then I think it should match in the comment.